thirdeye/thirdeye-pinot/src/main/resources/assets/javascript/HashParams.js

```diff
@@ -43,7 +51,6 @@ function HashParams() {
 HashParams.prototype = {
     init : function() {
```

Comment:
To be consistent with the other changes in this RB, please use `init() { ... }` here.